### ISSUE TITLE
Trigger table functionalities (download, filter) from outside source

### ISSUE THREAD

**Issue by Corouna** [gregnb/mui-datatables#972]
As titled,

Is this possible?

Like if i create a custom download button at the top of my page, how can i link it with the table's download callback so that it would trigger the same functionality like the table's download button?

Thanks

**Comment by gabrielliwerant** [comment]
I don't think this is possible at the moment. What would you be looking to do, just use different buttons/icons with the same actions?

**Comment by Corouna** [comment]
Yes @gabrielliwerant .. the table has all the features that my project wants.. just that its not aligning well with the design, particulary the buttons at the table toolbar. In my project design, the buttons to trigger download and filter are placed at the header of our application. 

But its okay if this is not possible. Wondering if there's other alternative that i can do for this?

**Comment by gabrielliwerant** [comment]
I understand, I think that's a legitimate issue, as there aren't a lot of good ways to customize this at the moment outside of creating an entirely new toolbar, but that suffers from being too much effort if you really just want to change the appearance of things and it's still limiting in terms of placement. So right now, I don't think you have any good alternatives.

I'm open to proposals for how to tackle this issue.
